mkdocs/docs/HPC/torque_frontend_via_jobcli.md

```diff
@@ -0,0 +1,131 @@
+# Torque frontend via jobcli
+
+## What is Torque
+
+[Torque](https://en.wikipedia.org/wiki/TORQUE) is a resource manager for submitting and managing jobs on an HPC cluster. It is and implementation of [PBS (Portable Batch System)](https://en.wikipedia.org/wiki/Portable_Batch_System). Torque isn't widely used anymore, so the {{hpcinfra}} no longer uses Torque in the backend, however the commands were kept to avoid researchers having to learn other commands.
+
+## Slurm backend
+
+Just like Torque, [Slurm](https://en.wikipedia.org/wiki/Slurm_Workload_Manager) is a resource manager for submitting and managing jobs on an HPC cluster. Currently, Slurm is the most popular workload manager on HPC systems worldwide, however it makes use of different commands that are less intuitive and user friendly than Torque/PBS.
+
+## jobcli
+
+Jobcli was developed to make it possible for the {{hpcinfra}} to use a Torque frontend and a Slurm backend. In addition to that, it adds some additional options for Torque commands.
+
+### help option
```

Comment:
This should be under a separate subsection: `### Additional options for Torque commands supported by jobcli`